projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,12 @@
+<button (click)="debugClick()">Print Debuginfo</button>
+<br />
+
+<br />
+<div *ngFor="let c of allColors; let i = index">
+    <geoengine-color-attribute-input [ngModel]="c" (ngModelChange)="rebuildColorMap(i, $event)"></geoengine-color-attribute-input>
+    <button mat-mini-fab (click)="removeColorTab(i)">-</button>
+</div>
+<br />
+Add a new color:
+<geoengine-color-attribute-input [ngModel]="newTab" (ngModelChange)="changeNewTab($event)"></geoengine-color-attribute-input>
```

Comment:
I guess it would look better if you only had the button that creates a new entry.
After this, the user can edit that entry anyway.

Reply:
New entry created at the bottom of the list with a white color 7b9aacac815d6eac4398b0466d4c9b74d043a732